Specs/Scene/ConditionsExpressionSpec.js

```diff
@@ -113,12 +113,40 @@ defineSuite([
         var expression = new ConditionsExpression(jsonExpWithExpression);
         expect(expression.evaluate(new MockFeature(101))).toEqual(Color.BLUE);
         expect(expression.evaluate(new MockFeature(52))).toEqual(Color.RED);
-        expect(expression.evaluate(new MockFeature(3))).toEqual(Color.GREEN);
+        expect(expression.evaluate(new MockFeature(3))).toEqual(Color.LIME);
     });
 
     it('evaluates undefined conditional expression', function() {
         var expression = new ConditionsExpression(jsonExpWithUndefinedExpression);
         expect(expression._expression).toEqual(undefined);
         expect(expression.evaluate(undefined)).toEqual(Color.BLUE);
     });
+
+    it('gets shader function', function() {
+        var expression = new ConditionsExpression(jsonExpWithExpression);
+        var shaderFunction = expression.getShaderFunction('getColor', '', {}, 'vec4');
+        var expected = 'vec4 getColor() \n' +
+                       '{ \n' +
+                       '    if (((Height / 2.0) > 50.0)) \n' +
+                       '    { \n' +
+                       '        return vec4(vec3(0.0, 0.0, 1.0), 1.0); \n' +
+                       '    } \n' +
+                       '    else if (((Height / 2.0) > 25.0)) \n' +
+                       '    { \n' +
+                       '        return vec4(vec3(1.0, 0.0, 0.0), 1.0); \n' +
+                       '    } \n' +
+                       '    else if (true) \n' +
+                       '    { \n' +
+                       '        return vec4(vec3(0.0, 1.0, 0.0), 1.0); \n' +
```

Comment:
I changed green to lime so that this check is easier, green would produce a vec3 like `vec3(0.0, 0.5019607843137255, 0.0)`